src/database/appConfig/appConfig.entity.ts

```diff
@@ -0,0 +1,11 @@
+import { Column, Entity } from 'typeorm';
+import { BaseEntity } from '../base.entity';
+
+@Entity()
+export class AppConfig extends BaseEntity {
```

Comment:
rename it to just `Config`. It is obvious that we talk about general configs in the app. There is no need to mention it explicitly